### PR TITLE
add index.docker.io/ prefix to image: in docs/example

### PR DESCRIPTION
**Pull request by vorburger**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it: 

https://wiki.archlinux.org/index.php/CRI-O#Testing Docker Hub is not hard-coded, registry prefix best explicit.

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:

Hope this makes sense.

#### Does this PR introduce a user-facing change? 

NONE

### CODE REVIEW

File: docs/examples/container-config.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "name": "busybox"
   },
   "image":{
-    "image": "busybox"
+    "image": "index.docker.io/busybox"
```
Comment (saschagrunert) [inline docs/examples/container-config.json:6]: The `index` subdomain is not required from my point of view: 
```suggestion
    "image": "docker.io/busybox"
```

Comment (feiskyer) [reply docs/examples/container-config.json:6]: @vorburger could you remove the `index` prefix?

Comment (vorburger) [reply docs/examples/container-config.json]: @feiskyer done. (Sorry it took a moment.) Merge?

**Comment by vorburger** [comment]
@saschagrunert @feiskyer @runcom merge?

File: docs/examples/container-config.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 metadata:
   name: busybox
 image:
-  image: busybox:latest
+  image: index.docker.io/busybox:latest
```
Comment (feiskyer) [inline docs/examples/container-config.yaml:4]: could you remove `index` here?

Comment (vorburger) [reply docs/examples/container-config.yaml]: @feiskyer done here as well, sorry it missed that 2nd one initially, and sorry it took a moment to get back here. Merge now?

File: docs/examples/container-config.json (hunk 1 above)
Comment (saschagrunert) [reply docs/examples/container-config.json:6]: It looks like it's still there :)

Comment (feiskyer) [inline docs/examples/container-config.json:6]: index is added back again?

Comment (vorburger) [reply docs/examples/container-config.json]: @feiskyer fixed (again); sorry - this is embarrassing - that's what you get when accepting Suggested Changes on GitHub's UI and forget to pull that locally - making further changes causes you to loose those fixes again. 😈 Merge now?

**Pull request merged**